style.css

```diff
@@ -457,3 +478,198 @@ textarea {
   color: #42526e;
   margin-bottom: 300px;
 }
+
+.d-desk {
+  display: none;
+}
+
+/* Desktop Version */
+
+@media screen and (min-width: 768px) {
+  #main-header {
+    display: none;
+  }
+
+  #main-section {
+    min-height: 120vh;
+    background-image: url(./img/Header-llustration-desktop.svg);
+    padding-top: 2%;
+    justify-content: start;
+    gap: 3%;
+  }
+
+  #main-section header {
+    display: block;
+    width: 100%;
+    margin-bottom: 12%;
+  }
+
+  .dis-dsk {
+    display: inline;
+  }
+
+  #main-section p {
+    width: 56.5%;
+  }
+
+  .f-nav {
+    font-weight: 550;
+    font-size: 15px;
+    line-height: 110%;
+    text-align: center;
+    color: #344563;
+  }
+
+  .inline-flex {
+    justify-content: space-between;
+    width: 18%;
+  }
+
+  .srlbtn {
+    margin-top: 4%;
+  }
+
+  /* Works Section */
+
+  #works-section {
+    margin-top: 8%;
+  }
+
+  .title::after {
+    width: 0;
+    height: 0;
+    background-color: none;
+    border-radius: none;
+    position: relative;
+  }
+
+  .title {
+    width: 100%;
+  }
+
+  #cards {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 2%;
+    justify-content: center;
+    margin: 50px 0 0 0;
+  }
+
+  /* about myself */
+
+  #about-myself {
+    background-image: url(./img/frame-des.svg);
+    margin-top: 5%;
+  }
+
+  .pAbout {
+    align-self: center;
+    padding-top: 2%;
+    padding-bottom: 2%;
+    width: 56.5%;
+  }
+
+  #about-myself .card-action {
+    width: 12%;
+  }
+
+  .bg-about {
+    background-image: url(./img/illust-about-desk.svg);
+    background-repeat: no-repeat;
+    background-size: unset;
+    background-position: left bottom 70px;
+    width: 100%;
+  }
+
+  #about-cards-con {
+    padding-top: 2%;
+    display: flex;
+    flex-direction: row;
+    gap: 2%;
+    width: 80%;
+  }
```

Comment:
- [ ] Your about cards are great and very similar to the Figma template, but they are not perfectly centered in the section on some smaller screens. Kindly adjust the styling of this element to make it centered on any screen size like the template.
 - Your page:
 - 
<img width="1292" alt="image" src="https://user-images.githubusercontent.com/61919617/221038868-9e2abe6a-87d4-4360-81a1-c65b0a5f4d9f.png">

 - Figma template:
 - 
<img width="1238" alt="image" src="https://user-images.githubusercontent.com/61919617/221039114-18fa76d5-536e-470f-97fb-27e621675e42.png">